### PR TITLE
ncm-metaconfig: add service lmod

### PR DESCRIPTION
**Pull request by stdweird**
New metaconfig service for site global configuration of [lmod](https://github.com/TACC/Lmod)

### CODE REVIEW

File: ncm-metaconfig/src/main/metaconfig/lmod/tests/profiles/default.pan [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+include 'metaconfig/lmod/config';
+
+prefix "/software/components/metaconfig/services/{/etc/lmodrc.lua}/contents/prop";
+"state/experimental" = dict(
+    "short", list("E"),
+    "long", list("E"),
+    "color", "blue",
+    "doc", "Experimental",
+    );
```
Comment (jrha) [inline ncm-metaconfig/src/main/metaconfig/lmod/tests/profiles/default.pan]: Extra indent at close of dict throughout this template.

File: ncm-metaconfig/src/test/perl/service-lmod.t [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    set_json_typed()
+}
+use Test::More;
+use Test::Quattor::TextRender::Metaconfig;
+
+ok(get_json_typed(), "json_typed enabled for lmod render");
+
+my $u = Test::Quattor::TextRender::Metaconfig->new(
+        service => 'lmod',
+    )->test();
```
Comment (jrha) [inline ncm-metaconfig/src/test/perl/service-lmod.t]: indentation here is weird too

**Comment by stdweird** [comment]
@jrha done

**Comment by jrha** [comment]
retest this please

**Pull request merged**